Ska/tdb/tdb.py

```diff
@@ -212,19 +309,27 @@ def find(self, *matches):
         """Find MSIDs with one or more ``matches`` in the ``msid``, ``description`` or
         ``technical_name``.
 
-        Examples::
+        Examples
+        --------
+
+        >>> msids.find('tephin')
+        [<MsidView msid="TEPHIN" technical_name="EPHIN SENSOR HOUSING TEMP">]
 
-          >>> msids.find('tephin')
-          [<MsidView msid="TEPHIN" technical_name="EPHIN SENSOR HOUSING TEMP">]
+        >>> msids.find('aca', 'filter')
+        [<MsidView msid="AOACIDPX" technical_name="ACA DATA PROCESSING DEFECTIVE PIXEL FILTER ENAB/DISA">,    # noqa
+        <MsidView msid="AOACIIRS" technical_name="ACA DATA PROCESSING IONIZING RADIATION FILTER ENAB/DISA">,  # noqa
+        <MsidView msid="AOACIMSS" technical_name="ACA DATA PROCESSING MULTIPLE STARS FILTER ENAB/DISA">,      # noqa
+        <MsidView msid="AOACISPX" technical_name="ACA DATA PROCESSING SATURATED PIXEL FILTER ENAB/DISA">]     # noqa
 
-          >>> msids.find('aca', 'filter')
-          [<MsidView msid="AOACIDPX" technical_name="ACA DATA PROCESSING DEFECTIVE PIXEL FILTER ENAB/DISA">,
-           <MsidView msid="AOACIIRS" technical_name="ACA DATA PROCESSING IONIZING RADIATION FILTER ENAB/DISA">,
-           <MsidView msid="AOACIMSS" technical_name="ACA DATA PROCESSING MULTIPLE STARS FILTER ENAB/DISA">,
-           <MsidView msid="AOACISPX" technical_name="ACA DATA PROCESSING SATURATED PIXEL FILTER ENAB/DISA">]
+        Parameters
+        ----------
+        matches:
```

Comment:
The thing that started it all... should this be `*matches` to indicate a one or more positional args? For me the comment below is not entirely clear to that point (I might try passing in a list of regexes).